stdlib/universe/universe.flux

```diff
@@ -283,11 +280,11 @@ _sortLimit = (n, desc, columns=["_value"], tables=<-) => tables
 
 // top sorts a table by columns and keeps only the top n records.
 top = (n, columns=["_value"], tables=<-) => tables
-    |> _sortLimit(n: n, columns: columns, desc: true)
```

Comment:
These changes should not be needed and an explicit import should not be needed. With your change to add the package to the prelude, I don't believe any changes to `universe` should be needed. I think putting it in the prelude list where it is is a pretty elegant solution.

Reply:
Made a small change to the way we infer the prelude and this now works.